servers/republik/modules/crowdfundings/lib/payments/stripe/deleteCustomer.js

```diff
@@ -0,0 +1,32 @@
+const getClients = require('./clients')
+
+module.exports = async ({
+  userId,
+  pgdb
+}) => {
+  const { accounts } = await getClients(pgdb)
+
+  const customers = await pgdb.public.stripeCustomers.find({
+    userId
+  })
+
+  if (!customers.length) {
+    return
+  }
+
+  return Promise.all(
+    customers.map(customer => {
+      const account = accounts.find(a => a.company.id === customer.companyId)
+      if (account) {
+        return Promise.all([
+          account.stripe.customers.del(customer.id),
```

Comment:
good catch! I changed it so that this works. Tested it with a "corrupted" stripe customer id, the error get's raised now. When I "fixed" the stripe customer id, and rerun `deleteUser`, the customer got deleted successfully.